### DISCUSSION TITLE
Enable or disable everything possible with API HTTP

### DISCUSSION THREAD

**Discussion by sagitaz** [blakeblackshear/frigate#12132] [Ask A Question] [answered]
### What is your question:

Hello,

I have created a plugin for the Jeedom home automation application based on the HTTP API.

Is there a solution to enable or disable :
- Snapshot
- Recording
- Detect
- Audio
- Motion

I don't want to use MQTT, because I want to provide a solution for users who don't use it.

Thank you very much

**Comment by NickM-27** [comment]
this is not currently supported

**Reply by KARMA-UA** [reply]
> this is not currently supported

Still not supported?
Thanks!

**Reply by NickM-27** [reply, marked as answer]
correct, there is a pinned feature request for it
